api/helpers/gamestate/validate-gamestate.js

```diff
@@ -33,8 +33,8 @@ function validatePhase(param) {
 }
 
 function validatePlayedBy(param) {
-  if (param === null || param === undefined || Number.isNaN(param) || param < 0 || param > 1) {
-    throw new Error('The playedBy attribute cannot be null and must be a number in [0,1]');
+  if (![null, 0, 1].includes(param)) {
```

Comment:
It makes the most sense for `playedBy` to be null for the first gamestate (initialize) since neither player "did" anything. That said alternatively we could say it's p1 because they are the "dealer"

Reply:
Update: opting to make it p1 the player who 'deals' and to rename the 'INITIALIZE` moveType to 'DEAL'